src/query/query_manager.c

```diff
@@ -2181,6 +2190,48 @@ qmgr_is_related_class_modified (THREAD_ENTRY * thread_p, XASL_CACHE_ENTRY * xasl
   return false;
 }
 
+QMGR_TRAN_STATUS
+qmgr_check_dblink_trans (THREAD_ENTRY * thread_p, int tran_index, bool is_abort)
+{
+  QMGR_TRAN_STATUS status = QMGR_TRAN_NULL;
+  QMGR_QUERY_ENTRY *query_p, *prev_query = NULL;
+  QMGR_TRAN_ENTRY *tran_entry_p;
+
+  tran_entry_p = &qmgr_Query_table.tran_entries_p[tran_index];
+  query_p = tran_entry_p->query_entry_list_p;
+
+  while (query_p)
+    {
+      if (query_p->xasl_ent && query_p->xasl_ent->one_clone.xasl && query_p->xasl_ent->one_clone.xasl->spec_list)
+	{
+	  if (query_p->xasl_ent->one_clone.xasl->spec_list->type == TARGET_DBLINK)
+	    {
+	      struct access_spec_node *spec = query_p->xasl_ent->one_clone.xasl->spec_list;
```

Comment:
```suggestion
	       ACCESS_SPEC_TYPE *spec = query_p->xasl_ent->one_clone.xasl->spec_list;
```
다음과 같이 수정해도 좋을 거 같습니다.